src/main/java/duke/UserInterface.java

```diff
@@ -0,0 +1,144 @@
+                    + "                              \\\\\n"
+                    + "                               \\\\\n"
+                    + "                                \\\n"
+                    + "                                 \\\n";
+
+    private static final String BORDER = "      ";
+
+    private static final String ERROR = "    ERROR: ";
+
+    private static final String NEWLINE = "\n";
+
+    /**
+     * To print out the task that is just added to the list.
+     * @param task Task of which to be added.
+     * @param totalNumber The prevailing number after a task is added.
+     * @return string to print whenever a task is added to the list.
+     */
+    public String printAddTask(String task, int totalNumber) {
+        String response = BORDER + "Steady! I add... wait ah.." + NEWLINE;
+        response += BORDER + BORDER + "ADDED: " + task + NEWLINE;
+        response += BORDER + "Now you got " + totalNumber + " tasks" + NEWLINE;
+        return response;
+    }
+
+    /**
+     * Preamble while the chat bot retrieves list of the tasks.
+     * @return response from when user prompts List Command
+     */
+    public String listTask() {
+        return BORDER + "Retrieving your list, patient ah!\n";
+    }
+
+    /**
+     * To print out the list of tasks in an ordered fashion.
+     * @param listNumber Index of the tasking in the list.
+     * @param task The details of the task itself.
+     * @return response from Duke to list Command
+     */
+    public String printTask(int listNumber, String task) {
+        String response = String.format("%2d. %s\n", listNumber, task);
+        return BORDER + response;
+    }
+
+    /**
+     * To print the preamble as well as the details of the task marked as done.
+     * @param task Task of which to be marked as done.
+     * @return response from Duke to print Done command
+     */
+    public String printDone(String task) {
+        String response = BORDER + "Swee la, task done liao:" + NEWLINE;
+        response += BORDER + BORDER + task + NEWLINE;
+        return response;
+    }
+
+    /**
+     * Preamble as well as printing of the sequence of deletion.
+     * @param task Task to be deleted.
+     * @param remaining Remaining number of tasks after deletion.
+     */
+    public String printDelete(String task, int remaining) {
+        String response = BORDER + "Delete liao boss:" + NEWLINE;
+        response += BORDER + "Remaining Tasks: " + remaining + NEWLINE;
+        return response;
+    }
+
+    /**
+     * Standard error message to be printed.
+     * @param errorMessage The details of the error message itself.
+     */
+    public String printError(String errorMessage) {
+        return ERROR + errorMessage + NEWLINE;
+    }
+
+    /**
+     * Preamble for the initialisation of the Pikachu Chatbot.
+     * @return response for initialisation and instruction for User.
+     */
+    public String welcomeMessage() {
```

Comment:
Should this be a verb since it is a method? maybe something like generateMessage?

Reply:
Thanks Chris took all your recommendation into account